ui/panels-plugin/src/plugins/gauge-chart/GaugeChartPanel.tsx

```diff
@@ -78,14 +85,38 @@ export function GaugeChartPanel(props: GaugeChartPanelProps) {
     },
   };
 
+  // no data message handled inside chart component
+  if (gaugeData.length === 0) {
+    return (
+      <GaugeChart
+        width={contentDimensions.width}
+        height={contentDimensions.height}
+        data={EMPTY_GAUGE_SERIES}
+        unit={unit}
+        axisLine={axisLine}
+        max={thresholdMax}
+      />
+    );
+  }
+
+  // show separate chart for each time series returned
+  const chartWidth = contentDimensions.width / gaugeData.length;
   return (
-    <GaugeChart
-      width={contentDimensions.width}
-      height={contentDimensions.height}
-      data={chartData}
-      unit={unit}
-      axisLine={axisLine}
-      max={thresholdMax}
-    />
+    <Stack direction="row">
+      {gaugeData.map((series, seriesIndex) => {
+        if (seriesIndex >= MAX_GAUGE_SERIES_DISPLAY) return null;
```

Comment:
Should we do anything to communicate to the user that additional information is getting cut off, especially in edit mode when creating the chart?

Reply:
+1

Reply:
Yeah I definitely agree, when not in edit mode, curious where/how do you think the warning should display? 

Also, given all the UX questions and potential panel options we may need to add, I'm going to leave this open for a bit until I can discuss further with @foxbat07 (other questions are around series name display, how to handle cases where the value does not fit, and should charts for each series wrap to multiple lines?)

Reply:
Made updates and now there is no longer a max series limit! Each chart has a minWidth and `overflowX: 'scroll'` is used so you can scroll horizontally to see any gauges that aren't visible (see [recording here](https://user-images.githubusercontent.com/9369625/206485429-c4c7c0b0-1a21-4f66-87ca-0388955d33fe.mp4))